WMFComponents/Sources/WMFComponents/Components/Activity Tab/WMFActivityTabView.swift

```diff
@@ -560,6 +560,12 @@ public struct WMFActivityTabView: View {
 
                     if index < categories.count - 1 {
```

Comment:
I think it could be that because you aren't showing a divider for the last item, some of your in-between spacing on line 554 is lost, so you aren't getting that consistent bottom padding for the last item. Here I am applying a blue background to the line 554 VStack:

<img width="559" height="1062" alt="Screenshot 2025-12-03 at 8 32 10 AM" src="https://github.com/user-attachments/assets/32174d88-3dd8-4436-826c-8b9a77feccdd" />